src/components/main/numbers/Numbers.js

```diff
@@ -0,0 +1,12 @@
+const createList = (props) => {
+  const arr = new Array(props.to - props.from + 1).fill().map((_, idx) => props.from + idx);
+  if (props.odd) {
+    return arr.filter(item => item % 2 === 0).map(item => <li key={item}>{item}</li>);
+  }
+  if (props.even) {
+    return arr.filter(item => item % 2 !== 0).map(item => <li key={item}>{item}</li>);
+  }
+  return arr.map(item => <li key={item}>{item}</li>);
+};
```

Comment:
`item => <li key={item}>{item}</li>` it is the same function for all cases, so it is good to have it defined in one place